models/yang/annotations/openconfig-mclag-annot.yang

```diff
@@ -0,0 +1,152 @@
+module openconfig-mclag-annot {
+    yang-version "1";
+
+    namespace "http://openconfig.net/yang/mclag-annot";
+    prefix "oc-mclag-annot";
+
+    import sonic-extensions { prefix sonic-ext; }
+    import openconfig-mclag { prefix oc-mclag; }
+
+    // meta
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "Openconfig MCLAG yang Annotations";
+
+    revision 2020-01-20 {
+        description
+            "Initial revision.";
+    }
+
+    deviation /oc-mclag:mclag/oc-mclag:mclag-domains/oc-mclag:mclag-domain {
+        deviate add {
+            sonic-ext:table-name "MCLAG_DOMAIN";
+        }
+    }
+
+    deviation /oc-mclag:mclag/oc-mclag:interfaces/oc-mclag:interface {
+        deviate add {
+            sonic-ext:table-name "MCLAG_INTERFACE";
```

Comment:
The subtree transformer will cover both the YangToDb and DbToYang cases and table-name annotation may not be required here. What is the use case to add this table-name annotation here?

Reply:
table transfomer was added to map ocyang to sonic yang table name directly. But i believe subtree was added later and this is how currently working code is there.  As you said subtree would take care of it it is not required, since it is the same way currently it is there working code i prefer to keep it

Reply:
It's best to clean it up to prevent others from copying.

Reply:
Taken care of the comment, removed table-name annotation. please take a look